builds/misc/images.yaml

```diff
@@ -13,7 +13,11 @@ jobs:
     pool:
       vmImage: 'ubuntu-16.04'
     steps:
-      - template: templates/install-dotnet3.yaml
+      - task: UseDotNet@2
+        displayName: Install .NET Core sdk
+        inputs:
+          packageType: sdk
+          version: 3.0.100
```

Comment:
I think you want 3.1 because it's LTS; 3.0 is already end-of-life. I'd use `3.1.x` here.